### PR TITLE
Add kibana ansible role

### PR DESCRIPTION
**Pull request by bemillenium**

### CODE REVIEW

File: handlers/main.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+---
+- name: restart kibana
+  service: name=kibana state=restarted
```
Comment (winggundamth) [inline handlers/main.yml]: Please use our convention

```
  service:
    name: kibana
    state: restarted
```

File: meta/.galaxy_install_info [hunk 2]
```diff
@@ -0,0 +1 @@
+{install_date: 'Fri Jan  4 04:19:43 2019', version: master}
```
Comment (winggundamth) [inline meta/.galaxy_install_info]: I think this file should be removed

File: tasks/kibana_pre.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Ensure dependency is installed (Ubuntu)
+  apt: name=apt-transport-https state=present
```
Comment (winggundamth) [inline tasks/kibana_pre.yml]: Use our convention

```
  apt:
    name: apt-transport-https
    state: present
```

File: tasks/kibana_pre.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  
+- name: Add Elasticsearch apt key.
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch
+    state: present
+
+- name: Add Kibana repository.
+  apt_repository:
+    repo: 'deb https://artifacts.elastic.co/packages/{{ kibana_version }}/apt stable main'
```
Comment (winggundamth) [inline tasks/kibana_pre.yml]: We should use "deb https://artifacts.elastic.co/packages/{{ kibana_version }}/apt stable main" as variable in defaults/main.yml

File: tasks/kibana_pre.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Ensure dependency is installed (Ubuntu)
+  apt: name=apt-transport-https state=present
+  
+- name: Add Elasticsearch apt key.
+  apt_key:
+    url: https://artifacts.elastic.co/GPG-KEY-elasticsearch
```
Comment (winggundamth) [inline tasks/kibana_pre.yml]: We should use "https://artifacts.elastic.co/GPG-KEY-elasticsearch" as variable in defaults/main.yml

File: tasks/main.yml [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+- name: Include OS family specific variables
+  include_vars: "{{ item }}"
+  with_first_found:
+    - "{{ ansible_os_family }}.yml"
```
Comment (winggundamth) [inline tasks/main.yml]: Not sure if we don't have specific OS variable. Should we remove this and remove vars/Debian.yml as well

File: tasks/kibana_search_config.yml [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+    - kibana_config_check.results[0].stat is defined
+    - kibana_config_check.results[0].stat.isdir
+
+- name: Check if kibana template configuration file is exists
+  stat:
+    path: "{{ kibana_host_config_file_path }}"
+  become: false
+  delegate_to: 127.0.0.1
+  register: kibana_config_check
+- debug: var=kibana_config_check
```
Comment (winggundamth) [inline tasks/kibana_search_config.yml]: Remove debug

File: LICENSE [hunk 8]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Opsta
```
Comment (winggundamth) [inline LICENSE]: Copyright (c) 2019 Opsta

**Review by winggundamth — CHANGES_REQUESTED** [review]
Please check comment

File: handlers/main.yml (hunk 1 above)
Comment (bemillenium) [reply handlers/main.yml]: Done krub

File: tasks/kibana_pre.yml (hunk 3 above)
Comment (bemillenium) [reply tasks/kibana_pre.yml]: Done krub

File: tasks/kibana_pre.yml (hunk 4 above)
Comment (bemillenium) [reply tasks/kibana_pre.yml]: Done krub

File: meta/.galaxy_install_info (hunk 2 above)
Comment (bemillenium) [reply meta/.galaxy_install_info]: Done krub

File: tasks/kibana_pre.yml (hunk 5 above)
Comment (bemillenium) [reply tasks/kibana_pre.yml]: Done krub

File: tasks/main.yml (hunk 6 above)
Comment (bemillenium) [reply tasks/main.yml]: Done krub

File: tasks/kibana_search_config.yml (hunk 7 above)
Comment (bemillenium) [reply tasks/kibana_search_config.yml]: Done krub

File: LICENSE (hunk 8 above)
Comment (bemillenium) [reply LICENSE]: Done krub

**Review by winggundamth — APPROVED** [review]
LGTM

**Pull request merged**